web/src/components/workflow-editor/run-history/latest-workflow-run-status.tsx

```diff
@@ -0,0 +1,31 @@
+import { useListWorkflowRunsApi } from "@/hooks/use-list-workflow-runs-api";
+import { Flex, Text } from "@radix-ui/themes";
+import { useEffect, useState } from "react";
+import type { TWorkflowRunMetadata } from "@/types/workflow-runs";
+import ErrorCallout from "@/components/utils/error-callout";
+import WorkflowRunStatusIndicator from "./workflow-run-status-indicator";
+
+export function WorkflowRunStatus({ workflowId }: { workflowId: string }) {
+	const { data, isPending, error } = useListWorkflowRunsApi(workflowId, 1);
+	const [latestRun, setLatestRun] = useState<TWorkflowRunMetadata>();
+
+	useEffect(() => {
+		if (data && data.length > 0) {
+			setLatestRun(data[0]);
+		}
+	}, [data]);
+
+	if (isPending || !latestRun) {
+		return;
+	}
+
+	if (error) {
+		return <ErrorCallout />;
+	}
```

Comment:
Did you actually test how this looks? I think this would look really strange. I actually think in case of an error, we should just return `null` to not render anything.